foxpuppet/windows/base.py

```diff
@@ -1,56 +1,68 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Handles creation of a base object for interacting with Firefox windows."""
 
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.wait import WebDriverWait
 
 
 class BaseWindow(object):
+    """A base window model.
 
-    """
-        A base window model.
-
-        :param selenium: WebDriver object.
-        :type selenium:
-            :py:class:`~selenium.webdriver.remote.webdriver.WebDriver`
-        :param handle: A window handle.
-        :type handle: str
+    :param selenium: WebDriver object.
+    :type selenium:
+        :py:class:`~selenium.webdriver.remote.webdriver.WebDriver`
+    :param handle: A window handle.
+    :type handle: str
     """
 
     _document_element = (By.CSS_SELECTOR, ':root')
 
     def __init__(self, selenium, handle):
+        """Create a BaseWindow object.
+
+        Args:
+            selenium: (:obj:`Webdriver`): Firefox Webdriver object.
+            handle: (str): Webdriver Firefox window handle.
+        """
         self.selenium = selenium
         self.handle = handle
         self.wait = WebDriverWait(self.selenium, timeout=10)
 
     @property
     def document_element(self):
-        """ Returns the inner DOM window element.
+        """Return the inner DOM window element.
 
         :return: DOM window element.
         :return type:
             :py:class:`~selenium.webdriver.remote.webdriver.WebDriver` locator
-        """
 
+        Returns:
+            obj: Webdriver locator object for the DOM window element.
+
+        """
         return self.selenium.find_element(*self._document_element)
 
     @property
     def firefox_version(self):
-        """ Major version of Firefox in use.
+        """Major version of Firefox in use.
 
         :returns: Major component of the Firefox version.
         :rtype: int
+
+        Returns:
+            int: Major component of the Firefox version.
```

Comment:
Will sphinx understand lines 54-55? According to the docs it must be how liners 51-52 are. Maybe I am missing something.